service/controller/app/v1/resource/releasemigration/create.go

```diff
@@ -30,12 +30,21 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 		return microerror.Mask(err)
 	}
 
-	hasConfigMap, err := r.hasHelmV2ConfigMaps(ctx, cc.Clients.K8s.K8sClient(), key.ReleaseName(cr))
+	var tillerNamespace string
+	{
+		if key.InCluster(cr) {
```

Comment:
@rossf7 Would you mind to take a look again? or is it fine?

Reply:
Yes I think this is the best option and I did the same in my PR.

Only nit would be to use the metav1.NamespaceSystem constant but that's just my OCD.